### DISCUSSION TITLE
Clarification on the output units for SFCRNOFF

### DISCUSSION THREAD

**Discussion by psbecker** [NCAR/hrldas#224] [Q&A] [answered]
I understand that SFCRNOFF is the accumulated runoff but wanted to clarify, does it give accumulated runoff in mm/s or in mm/time step?
For instance, if I have the time step at hourly and the output files at hourly, is the SFCRNOFF given to me in mm/hour?
Thanks

**Comment by cenlinhe** [comment, marked as answer]
The SFCRNOFF is a continuously accumulated quantity with a unit of mm from your starting point of simulations, that is you need to use the 2nd hour SFCRNOFF output minus the 1st hour SFCRNOFF output to get the total SFCRNOFF accumulated in the past hour which has a unit of mm/hour.

**Reply by SmithSimmon** [reply]
I have one more question: if the model runs with an hourly time step but provides daily output, is the unit of SFCRNOFF using the second day output minus first day output  mm/day? Or do I need to multiply by 24 hours to convert it to a daily value

**Reply by cenlinhe** [reply]
In this case, the SFCRNOFF will be the total accumulated surface runoff (mm) from the starting point of your simulation to the specific output timepoint of that day. You will need to use the SFCRNOFF at that day's output time minus the SFCRNOFF at a prior day's output time to get the SFCRNOFF accumulated in the past day with a unit of mm/day.
